libres/lib/config/conf.cpp

```diff
@@ -1050,7 +1051,7 @@ bool conf_instance_get_path_error(const conf_instance_type *conf_instance) {
                 conf_item->conf_item_spec;
             if (conf_item_spec->dt == DT_FILE) {
                 if (!fs::exists(conf_item->value))
-                    path_errors = true;
+                    path_errors.insert(conf_item_spec->name);
```

Comment:
could this also include the name of the file it is trying load

Reply:
done